packages/eas-cli/src/credentials/android/utils/googleServiceAccountKey.ts

```diff
@@ -34,3 +38,44 @@ export function readAndValidateServiceAccountKey(keyJsonPath: string): GoogleSer
     throw err;
   }
 }
+
+export async function selectGoogleServiceAccountKeyAsync(
+  keys: GoogleServiceAccountKeyFragment[]
+): Promise<GoogleServiceAccountKeyFragment> {
+  const sortedKeys = sortGoogleServiceAccountKeysByUpdatedAtDesc(keys);
+  const { chosenKey } = await promptAsync({
+    type: 'select',
+    name: 'chosenKey',
+    message: 'Select a Google Service Account Key:',
+    choices: sortedKeys.map(key => ({
+      title: formatGoogleServiceAccountKey(key),
+      value: key,
+    })),
+  });
+  return chosenKey;
+}
+
+function sortGoogleServiceAccountKeysByUpdatedAtDesc(
+  keys: GoogleServiceAccountKeyFragment[]
+): GoogleServiceAccountKeyFragment[] {
+  return keys.sort(
+    (keyA, keyB) =>
+      new Date(keyB.updatedAt).getMilliseconds() - new Date(keyA.updatedAt).getMilliseconds()
```

Comment:
```suggestion
      new Date(keyB.updatedAt) - new Date(keyA.updatedAt)
```

<img width="398" alt="Screenshot 2021-09-29 at 10 21 27" src="https://user-images.githubusercontent.com/5256730/135230783-3f5a1182-190b-4f80-816c-316b9cc4da38.png">

Reply:
;)

Reply:
I think that using getTime is cleaner than relying on js to auto-convert date to number

Reply:
also the same mistake was already merged in https://github.com/expo/eas-cli/commit/4674cbebc5b99c38e6bd14e11b953e57e96e58fd

Reply:
Good catch! I've changed this to `getTime` because typescript errors when you perform arithmetic on Date objects. Though it is worth noting vanilla javascript auto-converts dates to a number in this case.

Reply:
This PR refactors https://github.com/expo/eas-cli/commit/4674cbebc5b99c38e6bd14e11b953e57e96e58fd as well, so fixing the mistake here is sufficient :)